docs/data-sources/accesszone.md

```diff
@@ -88,7 +88,21 @@ output "powerscale_accesszone" {
 
 Optional:
 
-- `names` (Set of String)
+- `alternate_system_provider` (Set of String) List of alternate_system_provider
+- `cache_entry_expiry` (Set of Number) List of cache_entry_expiry
+- `create_path` (Boolean) Value for create_path
+- `groupnet` (Set of String) List of groupnet
+- `home_directory_umask` (Set of Number) List of home_directory_umask
+- `id` (Set of String) List of id
+- `map_untrusted` (Set of String) List of map_untrusted
+- `name` (Set of String) List of name
```

Comment:
We are renaming a field here, which will break backward compatibility.
Instead, I suggest adding logic to convert plural names to singular names (remove the last `s` in list types) while filtering.

Reply:
We can deprecate attribute https://developer.hashicorp.com/terraform/plugin/framework/deprecations#provider-attribute-rename